src/frontend/flight/handler.rs

```diff
@@ -47,6 +48,8 @@ pub(super) struct SeafowlFlightHandler {
     pub context: Arc<SeafowlContext>,
     pub results: Arc<DashMap<String, Mutex<SendableRecordBatchStream>>>,
     sync_writer: Arc<RwLock<SeafowlDataSyncWriter>>,
+    // Denotes whether we're past the first sync reponse, thus indicating Seafowl (re)starts
+    first_sync: Arc<AtomicBool>,
```

Comment:
This doesn't need to be tracked for each origin separately?

Reply:
My understanding was no, the overall response order is important, cc @eatonphil?

Reply:
That's right